user_guide_src/source/installation/upgrade_4xx.rst

```diff
@@ -165,6 +165,23 @@ Hooks
 - The hook point ``post_system`` has moved just before sending the final rendered
   page.
 
+Error Handling
+==============
+
+- The behavior in CI4 has been slightly changed.
```

Comment:
At first glance, I don't quite understand it well. Can we make it in tabular form?